codegen/proto/src/lib.rs

```diff
@@ -40,6 +40,179 @@ pub mod moby {
     pub mod filesync {
         pub mod v1 {
             include!("generated/moby.filesync.v1.rs");
+            
```

Comment:
Could we move this part of the code into another file and `include!` it here - simply makes it easier for me maintaining this file as it sometimes gets new generated files.

Reply:
I changed things around a bit and everything is now incorporated into the `codegen/proto` infrastructure! That documents the process in a way and also gives the new code a home (which I added using `include!`).